samples/cpp/epipolar_lines.cpp

```diff
@@ -1,107 +0,0 @@
-// This file is part of OpenCV project.
```

Comment:
I propose to move it to snippets and add reference for findFundamentalMat at least. There is something abstract. I propose to replace it.